src/sentry/types/integrations.py

```diff
@@ -52,6 +52,10 @@ class ExternalProviderEnum(Enum):
     ExternalProviderEnum.CUSTOM: ExternalProviders.CUSTOM,
 }
 
+
+EXTERNAL_PROVIDERS_REVERSE_VALUES = {k.value: v for k, v in EXTERNAL_PROVIDERS_REVERSE.items()}
```

Comment:
i also needed to update the old types b/c currently `sentry` will use this, while the pr i mentioned in the description will use the symbols in the  file under integrations folder.